internal/wasi/wasi.go

```diff
@@ -775,21 +944,6 @@ func (a *wasiAPI) fd_read(ctx wasm.ModuleContext, fd uint32, iovsPtr uint32, iov
 	return wasi.ErrnoSuccess
 }
 
-func (a *wasiAPI) fd_close(ctx wasm.ModuleContext, fd uint32) wasi.Errno {
```

Comment:
keep definition in the same order as IIRC I had re-ordered these to match spec order. not a huge deal, but better to be consistent

Reply:
Actually it wat not, but I've fixed as discussed. Thanks!